.github/workflows/sanity_checks.yml

```diff
@@ -154,6 +154,12 @@ jobs:
           brew install ninja
           python3 -m pip install --pre meson
 
+      # The pkg-config file for openssl is borked: the libraries search
```

Comment:
I think disable openssl in ci_config.json with this comment.

Reply:
Rather than fix it for everybody?

Reply:
If we wait a couple of days, maybe it won't be an issue anymore: https://github.com/actions/runner-images/issues/9755#issuecomment-2081579531.

Reply:
sure. I don't know if editing sanity_checks.yml for a specific Homebrew bug is the correct solution here.

Reply:
So it's still not fixed…

I disagree that disabling openssl in `ci_config.json` is the right approach. It's a dependency of at least 8 wraps, IMHO fixing it in `sanity_checks.yml` so all wraps benefit from it is the better solution.

Alternatively, ignore the macOS arm64 failure and merge.

Reply:
Yeah why not. It's experimental anyways.